### PR TITLE
Modernization 3

### PR DESCRIPTION
**Pull request by phryneas**

### CODE REVIEW

File: src/language-server/project/client.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ import {
   DiagnosticSeverity,
 } from "vscode-languageserver/node";
 import LZString from "lz-string";
-import { stringifyUrl } from "query-string";
```
Comment (phryneas) [inline src/language-server/project/client.ts:31]: New versions of `"query-string"` are ESM only and I really don't want to touch the build process of the Extension right now.

File: src/language-server/project/client.ts [hunk 2]
```diff
@@ -412,7 +412,7 @@ export class GraphQLClientProject extends GraphQLProject {
                     this.frontendUrlRoot ?? "https://studio.apollographql.com";
 
                   const variant = this.config.variant;
-                  const graphId = this.config.graph;
+                  const graphId = this.studioGraphId;
```
Comment (phryneas) [inline src/language-server/project/client.ts:415]: If I have a config file like
```
module.exports = {
  client: {
    service: {
      name: "localSchema",
      localSchemaFile: "./starwarsSchema.graphql",
    },
  },
};
```
`graphId` would be `localSchema` - of course, linking to studio for that is kinda useless, but that's exactly what the extension did until now.

File: .changeset/five-houses-doubt.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"vscode-apollo": patch
+---
+
+Only show "Run in Studio" gutter action for local graphs if an endpoint is configured.
```
Comment (phryneas) [inline .changeset/five-houses-doubt.md:5]: I'm not 100% sure.. it seems like a good idea, but might someone get use from a link to explorer that doesn't fill in the endpoint?
My gut says removing this is the right thing, but I'd love a second opinion.

**Review by jerelmiller — APPROVED** [review]
💯

**Pull request merged**